docs/docs/getting-started/examples/filestash-s3proxy.md

```diff
@@ -69,3 +92,19 @@ Not adding the CA certificate will result in TLS authentication errors.
 
 Afterward, you can use a port forward to access the Filestash pod:
 `kubectl port-forward pod/$(kubectl get pod --selector='app=filestash' -o=jsonpath='{.items[*].metadata.name}') 8334:8334`
+
+4. After browsing to `localhost:8443`, Filestash will ask you to set an administrator password.
+After setting it you can directly leave the admin area by clicking the blue cloud symbol in the top left corner.
+Subsequently, you can select S3 as storage backend and enter your credentials.
+This will bring you to an overview of your buckets.
+If you want to deploy Filestash in production, please take a look at it's [documentation](https://www.filestash.app/docs/).
```

Comment:
```suggestion
If you want to deploy Filestash in production, take a look at its [documentation](https://www.filestash.app/docs/).
```
ms style guide "Avoid please except in situations where the customer is asked to do something inconvenient or the application or site is to blame for the situation."